components/unstoppable_domains/DEPS

```diff
@@ -0,0 +1,4 @@
+include_rules = [
```

Comment:
I guess this file will need more entries to pass `checkdeps`. For instance `components/security_interstitials` and everything else

Reply:
checkdeps passed tho, I was guessing maybe we whitelist things like components/* because I didn't see any error?

Reply:
Entries I added here are the ones I got error from running it.

Reply:
ah, right, there is a TODO in brave/components/DEPS to narrow down its scope.
Normally we should explicitly declare everything, especially intercomponent DEPS to avoid loops

Reply:
ok got it, I'll update the list to explicitly declare everything.

Reply:
Updated in https://github.com/brave/brave-core/pull/7702/commits/c22eb93c897462705331ec41bfc9bc6a1b4ae9ba